https://issues.redhat.com/browse/ACM-18209 Fix descr (#7576)
open-cluster-management/rhacm-docs: clusters/cluster_lifecycle/create_clusterset.adoc

[#managedclustersets_global]

> After installing {mce-short}, the following are automatically created on the hub cluster to ease management. Managed clusters that you create belong to the `global` `ManagedClusterSet` by default:

Reviewer: I would put the last sentence somewhere else ("Managed clusters that you create belong to the `global` `ManagedClusterSet` by default"). It's distracting that the intro sentence of the list is interrupted with another sentence that doesn't really fit there. I'd either add this to the first item on the list, which is currently on line 16 or add it to line 8.

```suggestion
After installing {mce-short}, the following resources are automatically created on the hub cluster to ease management:
```
